.changeset/swift-paths-change.md

```diff
@@ -0,0 +1,8 @@
+---
+"@snapwp/blocks": patch
+"@snapwp/types": patch
+"@snapwp/core": patch
+"@snapwp/next": patch
+---
+
+Update jsx compiler
```

Comment:
1. Conventional Commits
2. What the PR actually does. (Not sure how this relates to jsx compiling).


```suggestion
chore: Enforce "import/default" eslint ruleset.
```